shell/platform/android/io/flutter/embedding/android/FlutterFragment.java

```diff
@@ -326,6 +341,18 @@ protected String getDartEntrypointFunctionName() {
     return getArguments().getString(ARG_DART_ENTRYPOINT, "main");
   }
 
+  /**
+   * Returns the desired {@link FlutterView.RenderMode} for the {@link FlutterView} displayed in
+   * this {@code FlutterFragment}.
+   *
+   * Defaults to {@link FlutterView.RenderMode#surface}.
+   */
+  @NonNull
+  protected FlutterView.RenderMode getRenderMode() {
+    String renderModeName = getArguments().getString(ARG_FLUTTERVIEW_RENDER_MODE, null);
```

Comment:
subjective, nit: If you set the default value here to `FlutterView.RenderMode.surface.toString()` you could have the next statement consistently return without needing a null check and ternary.

Reply:
done